synapse/lib/modelrev.py

```diff
@@ -880,7 +884,7 @@ async def save():
             if nodeedits:
                 await save()
 
-    async def _normFormSubs(self, layers, formname, liftprop=None):
+    async def _normFormSubs(self, layers, formname, liftprop=None, cmprvalu=None, cmpr='='):
```

Comment:
Can we use `cmprvalu=s_common.novalu` as the sentinel for an empty argument here?

If we ever have to look at migrating a `data` type with the `cmprvalu` of `None` we have to rewrite the logic of the function.

Reply:
good catch